### DISCUSSION TITLE
Welcome to pocketpal-ai Discussions!

### DISCUSSION THREAD

**Discussion by a-ghorbani** [a-ghorbani/pocketpal-ai#82] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avatar21** [comment]
Wonder whats the recommended iPhone hardware and llm model require to run relatively well (usable code generation tool)? I'm using llama 3.2 1b with metal turned on, seems not very accurate or contextually kinda short to make it usable even for a small to medium problem. Like to hear some other opinion / actual feedback.

**Reply by deekzzz** [reply]
iPhone 15+ for 8gb

**Comment by jmlb** [comment]
is there any support for vision. I can successfully load a multimodal model like `Eris_PrimeV4-Vision-32k-7B-GGUF-IQ-Imatrix` and have a regular conversation without bug. But there does not seem to be any way to attach a photo to the conversation.

**Comment by saiidrahman** [comment]
سلام جناب اسغر قربانی من اپ Edge Gallery  را استفاده میکنم که از GPU مبایل استفاده میکند ولی PocketPal ai  تنها از cpu استفاده میکند که بسیار Slow است و این هم از مشخصات gpu در سکرینشات 
![Screenshot_20260410_165036_DevInfo](https://github.com/user-attachments/assets/9534ef06-ff18-4ae0-9589-1ad8797e55f9)
![Screenshot_20260410_165028_DevInfo](https://github.com/user-attachments/assets/49412099-fbed-4727-8ed0-4954e8805b22)
My phone is Samsung Note10 plus exynos cpu Mali gpu لطفا قابلیت استفاده از gpu mali-g76 را اضافه کن
